Lib/fontTools/ttLib/tables/D__e_b_g.py

```diff
@@ -0,0 +1,16 @@
+from . import DefaultTable
+import json
+
+class table_D__e_b_g(DefaultTable.DefaultTable):
+
+	def decompile(self, data, ttFont):
+		self.data = json.loads(data)
+
+	def compile(self, ttFont):
+		return str.encode(json.dumps(self.data))
+
+	def toXML(self, writer, ttFont):
+		writer.writecdata(self.compile(ttFont).decode())
```

Comment:
why the self.compile and the extra encode/decode step? Can't you just do `writer.writecdata(json.dumps(self.data))`?